drivers/filesystems/btrfs/volume.c

```diff
@@ -877,8 +812,9 @@ NTSTATUS vol_device_control(IN PDEVICE_OBJECT DeviceObject, IN PIRP Irp) {
             return vol_get_disk_extents(vde, Irp);
 
         default: { // pass ioctl through if only one child device
-            ULONG code = IrpSp->Parameters.DeviceIoControl.IoControlCode;
             NTSTATUS Status = vol_ioctl_passthrough(vde, Irp);
+#ifdef _DEBUG
+            ULONG code = IrpSp->Parameters.DeviceIoControl.IoControlCode;
 
 #ifdef __REACTOS__
```

Comment:
It looks that this ifdef block could be removed now...

Reply:
indeed

Reply:
changed in e720186ed8beb765c75222249dfe7e6d78bb3ab8